assets/js/projects.json

```diff
@@ -2,115 +2,115 @@
   {
     "title": "Reece",
     "designer": "Nora Warschewski",
-    "url": "../reece/",
+    "url": "/reece/",
```

Comment:
Changes in this file are for consistency and to help make writing [lines 44-48](https://github.com/norawarschewski/tw-2023/pull/7/files#diff-3708caf1ae0a3f4f266bc2060af1279ef61b739abdf1f2cd852049252f40a09cR44-R48) in the script a bit clearer.